criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
```

Comment:
I agree, but I at the moment there's quite a bit of chaos in NL about which software catalogs exist. It would be nice if the FFPC could provide resources of which catalogs may be relevant and even actively organize these catalogs.

Reply:
Yes, I agree that we should have a list of catalogs. That list might not quite fit in the catalog itself though, but perhaps a pointer to it. 

We are indeed planning a secondary resource where we provide a lot more examples of how the requirements in the standard could be met and possibly even linking to particularly elegant solutions.

Reply:
I also thinking the other way around. Software catalogs should only have to know where to find repositories for indexation. If we already have the `publiccode.yaml` criterion we don't need to enlist all repositories to catalogs but insteads should be able to only submit a (organizational) github, bitbucket, gitlab etc. account from where the catalogs can start there indexation. That makes findability a organizational responsability instead of a repository maintainer one.

Reply:
There is no incentive for catalog owners to change there processes when we adopt this criterium. However, when we can create an apt like structure the incentive is clear.